collect_app/src/main/res/layout/form_chooser_list.xml

```diff
@@ -18,19 +18,13 @@ the License.
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
-    <include layout="@layout/toolbar"/>
-
-    <include
-            layout="@layout/toolbar_action_bar_shadow"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_below="@id/toolbar"/>
+    <include layout="@layout/app_bar_layout"/>
 
     <LinearLayout
             android:id="@+id/llParent"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:layout_below="@id/toolbar"
+            android:layout_below="@id/appBarLayout"
```

Comment:
We don't actually have this written down anywhere but generally my opinion is that layout IDs should be snakecased (`app_bar_layout`). I've never actually been able to find any good arbitrary opinions on this. Thoughts @lognaturel?